tests/testthat/_snaps/check_pkg_installed.md

```diff
@@ -6,6 +6,6 @@
       })
       user_facing_fn()
     Condition
-      Error in `user_facing_fn()`:
+      Error in `check_pkg_installed()`:
```

Comment:
Note to self: I think this is meant to remain `user_facing_fn()`. Need to look into it.

Reply:
Fixed.